corgi/core/migrations/0107_copy_srpm_license_data_to_binary_rpms.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 3.2.20 on 2023-11-27 05:50
+
+from django.db import migrations
+from corgi.core.models import Component
+
+
+def copy_srpm_license_data_to_binary_rpms(apps, schema_editor) -> None:
+    for scanned_srpm in (
+        Component.objects.filter(type=Component.Type.RPM, arch="src")
+        .exclude(license_concluded_raw="")
+        .iterator()
+    ):
+        for binary_rpm in scanned_srpm.provides.filter(type=Component.Type.RPM).exclude(
+            license_concluded_raw=""
+        ):
+            binary_rpm.openlcs_scan_url = scanned_srpm.openlcs_scan_url
+            binary_rpm.openlcs_scan_version = scanned_srpm.openlcs_scan_version
+            binary_rpm.license_concluded_raw = scanned_srpm.license_concluded_raw
+            binary_rpm.license_declared_raw = scanned_srpm.license_declared_raw
+            binary_rpm.copyright_text = scanned_srpm.copyright_text
+            binary_rpm.save()
```

Comment:
We could also use .update() on the queryset instead of setting properties on each binary RPM individually, which should be faster (fewer DB queries). Typing from memory, but I think something like below will work:
```suggestion
        scanned_srpm.provides.filter(
            type=Component.Type.RPM,
            license_concluded_raw="",
        ).update(
            copyright_text=Value(scanned_srpm.copyright_text)
            license_concluded_raw=Value(scanned_srpm.license_concluded_raw)
            license_declared_raw=Value(scanned_srpm.license_declared_raw)
            openlcs_scan_url=Value(scanned_srpm.openlcs_scan_url)
            openlcs_scan_version=Value(scanned_srpm.openlcs_scan_version)
            )
```